tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -456,3 +490,30 @@ _handle_ingress_nodeport() {
 	# By exporting this variable the kbs deploy script will install the nodeport service
 	export DEPLOYMENT_DIR=nodeport
 }
+
+
+# Prepare necessary resources for qemu-se runtime
+# Documentation: https://github.com/confidential-containers/trustee/tree/main/attestation-service/verifier/src/se
+prepare_credentials_for_qemu_se() {
+	echo "::group::Prepare credentials for qemu-se runtime"
+	config_file_path="/opt/kata/share/defaults/kata-containers/configuration-qemu-se.toml"
+	kata_base_dir=$(dirname $(kata-runtime --config ${config_file_path} env --json | jq -r '.Kernel.Path'))
+	if [ ! -d ${HKD_PATH} ]; then
+		>&2 echo "ERROR: HKD_PATH is not set"
+		return 1
+	fi
+	ibmse_creds_dir=$(mktemp -d -t ibmse.creds.XXXXXXXXXX)
```

Comment:
if you end up moving this code close to the tests then you will be able to use bats' temporary directory and not care about cleaning that directory later.

Reply:
A lifecycle of this directory should match with that of KBS rather than the tests. It is mounted to the KBS as a persistent volume and cleaned up by the post-action script. If it is removed at a task `running tests`, this would be problematic for uninstalling a cluster. I would like to leave it as it is. Thanks.

Reply:
In line with my argument, I took out the declaration of `IBM_SE_CREDS_DIR`. The env variable should be defined/cleaned up in the workflow for the internal SE k8s tests.

Reply:
I made a new function `repack_secure_image()` general to cover a case where a secure image only needs to be built without KBS. Thanks!